Vostok.Metrics.System/Host/NativeHostMetricsCollector_Linux.cs

```diff
@@ -13,21 +13,31 @@ internal class NativeHostMetricsCollector_Linux : IDisposable
         private readonly Regex pidRegex = new Regex("[0-9]+$", RegexOptions.Compiled);
```

Comment:
A piggyback comment that should have appeared in the previous review iteration: `ReadPerformanceInfo` contains a race (enumeration of process' file descriptors may fail due to process having already exited). Pls fix.